src/Tribe/Rewrite.php

```diff
@@ -407,19 +407,25 @@ public function get_canonical_url( $url, $force = false ) {
 		$query         = (string) parse_url( $url, PHP_URL_QUERY );
 		wp_parse_str( $query, $query_vars );
 
-		// Remove the `paged` query var if it's 1.
 		if ( isset( $query_vars['paged'] ) && 1 === (int) $query_vars['paged'] ) {
+			// Remove the `paged` query var if it's 1.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 2266e84020e4ee103743c0a47bfc538e</sub>